### PR TITLE
Added ProductName and FileDescription to example in README.md

### PR DESCRIPTION
**Pull request by pracucci**
I added `ProductName` and `FileDescription` to the example. It took 1 hour to realize I needed to exiting these 2 strings in order to completely rename the .exe file. Hope will easier life to other people as well.

### CODE REVIEW

**Comment by unindented** [comment]
@pracucci Sorry for the late response man. For whatever reason I didn't get notified of the PR...

I assumed people were familiar with the `VERSIONINFO` field: https://msdn.microsoft.com/en-us/library/windows/desktop/aa381058(v=vs.85).aspx

Should we include all possible fields somewhere in the `README`, along with their description, so that people have an easier time consuming this library?

They'd be:
- **Comments**: Additional information that should be displayed for diagnostic purposes.
- **CompanyName**: Company that produced the file, for example "Microsoft Corporation" or "Standard Microsystems Corporation, Inc.".
- **FileDescription**: File description to be presented to users. This string may be displayed in a list box when the user is choosing files to install?for example, "Keyboard Driver for AT-Style Keyboards".
- **FileVersion**: Version number of the file?for example, "3.10" or "5.00.RC2".
- **InternalName**: Internal name of the file, if one exists?for example, a module name if the file is a dynamic-link library. If the file has no internal name, this string should be the original filename, without extension.
- **LegalCopyright**: Copyright notices that apply to the file. This should include the full text of all notices, legal symbols, copyright dates, and so on.
- **LegalTrademarks**: Trademarks and registered trademarks that apply to the file. This should include the full text of all notices, legal symbols, trademark numbers, and so on.
- **OriginalFilename**: Original name of the file, not including a path. This information enables an application to determine whether a file has been renamed by a user. The format of the name depends on the file system for which the file was created.
- **ProductName**: Name of the product with which the file is distributed. This string is required.
- **ProductVersion**: Version of the product with which the file is distributed?for example, "3.10" or "5.00.RC2".

Would you mind adding them to your PR, and squashing all commits into one?

**Comment by pracucci** [comment]
Thanks for your feedback. I updated the README: you can get a [preview here](https://github.com/pracucci/grunt-rcedit). Feel free to give me any other feedback, in case there's any improvement I can do.

File: .gitignore [hunk 1]
```diff
@@ -1 +1,2 @@
 /node_modules/
+.DS_Store
```
Comment (unindented) [inline .gitignore]: Can you remove this from the project's `.gitignore`? It shouldn't be here, but rather in people's global `~/.gitignore`.

**Comment by unindented** [comment]
Only left a comment to remove a thing from `.gitignore`, cause it doesn't belong there. The rest is awesome.

Could you squash all your commits into one, to avoid polluting the commit history?

**Comment by pracucci** [comment]
Done. Thanks for the feedback.

**Comment by unindented** [comment]
Thank you very much for your help! 🎉

**Pull request merged**